src/QMCTools/PyscfToQmcpack_Spline.py

```diff
@@ -0,0 +1,947 @@
+######################################################################################
+## This file is distributed under the University of Illinois/NCSA Open Source License.
+## See LICENSE file in top directory for details.
+##
+## Copyright (c) 2016 Jeongnim Kim and QMCPACK developers.
+##
+## File developed by: Thomas Applencourt, applencourt@anl.gov,  Argonne National Laboratory
+##                    Hyeondeok Shin, hshin@anl.gov, Argonne National Laboratory
+##                    Yubo "Paul" Yang, yubo.paul.yang@gmail.com, University of Illinois Urbana-Champaign
+##                    Anouar Benali, benali@anl.gov, Argonne National Laboratory
+##                  
+## File created by: Thomas Applencourt, applencourt@anl.gov, Argonne National Laboratory
+#######################################################################################
+
+from __future__ import print_function
```

Comment:
If this is python 3 only (as a check below indicates), this import is not needed.

Reply:
You right, let me fix that